### PR TITLE
feat: give component EntityMembersListCard a memberType prop

### PR DESCRIPTION
**Pull request by djamaile**
Signed-off-by: djamaile <rdjamaile@gmail.com>

## Hey, I just made a Pull Request!

I wanted to use this component when showing the contributors of a certain Gitlab repo. In that context having members doesn't really make sense because a contributor doesn't have to be a member of the repo. Another use case I can see for this is companies having their own specific name for their engineers. 

#### :heavy_check_mark: Checklist

- [x ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ x] Tests for new functionality and regression tests for bug fixes
- [x ] Screenshots attached (for UI changes)
- [ x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

screencapture-localhost-3000-catalog-default-group-team-a-2022-01-08-19_40_22

### CODE REVIEW

**Comment by djamaile** [comment]
Hi! Was wondering the core maintainers even want this change in? If not then I know I have to copy the component to my internal repo and just make some tweaks to it. I completely understand if this change is totally different from the vision the maintainers have! Just let me know :)

**Comment by benjdlambert** [comment]
Sorry this was on my list of stuff to do today!

So when would this actually be used? Is this when you call your staff something different other then employees or members or something and want a new title for the members of a squad that applies company wide or a different use case?

**Comment by djamaile** [comment]
> Sorry this was on my list of stuff to do today!
> 
> So when would this actually be used? Is this when you call your staff something different other then employees or members or something and want a new title for the members of a squad that applies company wide or a different use case?

Yes that is exactly the use case. I am using this component as well in the servicePage and instead of calling them members I want to call them contributors. Because 'members' means something else in the context of Gitlab.

**Comment by benjdlambert** [comment]
Perfect - glad we're on the same level, so maybe something like `memberDisplayTitle` or `memberTitle` would be better than type as it's purely for visual display?

**Comment by djamaile** [comment]
@benjdlambert updated 😄

File: .changeset/gold-birds-happen.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/plugin-org': patch
+---
+
+For the component `EntityMembersListCard` you can now specify the type of members you have in a group. For example:
+
+```tsx
+<Grid item xs={12}>
+  <EntityMembersListCard memberType="Ninja's" />
```
Comment (adamdmharvey) [inline .changeset/gold-birds-happen.md:9]: given the change, think the example in the changeset needs an update?  `memberDisplayTitle`?

**Comment by awanlin** [comment]
@djamaile just wondering if maybe this could be helpful to you? They have a `ContributorsCard` in the GitLab plugin:

 https://github.com/loblaw-sre/backstage-plugin-gitlab/blob/bb8b1042c473cdeb65606d8af42e4f02cee00151/src/components/widgets/ContributorsCard/ContributorsCard.tsx#L21

File: .changeset/gold-birds-happen.md (hunk 1 above)
Comment (djamaile) [reply .changeset/gold-birds-happen.md]: Nice catch!

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Thanks! :pray:

**Pull request merged**